Dockerfile

```
@@ -1,29 +1,37 @@
 FROM python:3.9
 
-RUN SNIPPET="export PROMPT_COMMAND='history -a' && export HISTFILE=/commandhistory/.bash_history" && echo $SNIPPET >> "/root/.bashrc"
+ARG USERNAME=dev
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
 
 ENV LANG=C.UTF-8 \
   LC_ALL=C.UTF-8 \
   PATH="${PATH}:/root/.poetry/bin"
 
+RUN apt update -y && apt install -y sudo
+RUN groupadd --gid $USER_GID $USERNAME &&\
+ useradd --uid $USER_UID --gid $USER_GID -m $USERNAME &&\  
+ echo ${USERNAME} ALL=\(root\) NOPASSWD:ALL > /etc/sudoers.d/${USERNAME} &&\
+ chmod 0440 /etc/sudoers.d/${USERNAME}
```

```suggestion
 chmod 0440 /etc/sudoers.d/${USERNAME} && chsh ${USERNAME} -s /bin/bash
```

This makes `bash` the default shell for the `dev` user. That way keyboard shortcuts still work.

---

Fixed this change.  The only difference is I put chsh on its own line so it matches the rest of the command. This is definitely a nice addition.  Thanks for figuring out how to do it!

---

@jbencook do you have a preference on resolving conversations?  I think it is best to have the person that opened the suggestion resolve but wanted to double check your thoughts on that.

---

Hm yeah that's a good question -- I've usually done the reverse (the person who addresses the comment resolves it), but I definitely see what you mean and I don't have a strong preference either way